### PR TITLE
🚑️Rclone: downgrade to 1.63.1

### PR DESCRIPTION
**Pull request by sanderegg**
## What do these changes do?
After the tests done in https://github.com/ITISFoundation/osparc-simcore/discussions/5828 it is pretty visible that the new version of RClone, beside having a very bad bug that makes EC2 instances kernel panic that the current parameters make the download very slow, very bad.

Therefore this PR aims at downgrading the RClone back to 1.63.1 version, which seemed to be more stable even though still much slower (2x) than using official AWS tools. But at least it seems more or less bugfree.

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py [hunk 1]
```diff
@@ -192,8 +189,6 @@ async def store_to_s3(  # pylint:disable=too-many-locals,too-many-arguments
         f"{source_dir}",
         f"dst:{s3_path}",
         "--verbose",
-        "--multi-thread-streams",
```
Comment (GitHK) [inline services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py:195]: did you not try to remove the buffer-size and use-mmap to see if ti gets any increments compare to the was CLI tools?

**Review by GitHK — COMMENTED** [review]
👍

**Review by YuryHrytsuk — APPROVED** [review]

File: services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py (hunk 1 above)
Comment (sanderegg) [reply services/agent/src/simcore_service_agent/modules/volumes_cleanup/_s3.py:195]: yes I did, please check the discussion linked above. it makes 15 seconds difference, aka no changes at all.
Sorry but this tool just does not compare well with aws CLI.

**Review by matusdrobuliak66 — APPROVED** [review]
Thanks!

**Pull request merged**